### PR TITLE
Adding Tag Cloud Page.

### PR DESCRIPTION
**Pull request by xRahul**
I have added the tag cloud from my website to the theme. I'm just not sure where we should put the link for this. Let me know if you have any thoughts. Also, I have not added the hash svg icon here, so if you have a source for them, we can use that too.

Issue: https://github.com/nielsenramon/chalk/issues/128
Example: https://rahulja.in/tags

### CODE REVIEW

File: tags.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+description: "Tag cloud from all the posts"
+permalink: /tags
+sitemap: false
+---
+
+<div class="tags-page text-center">
+  <h1 class="tags-page-header">
+    Tags
+  </h1>
+
```
Comment (nielsenramon) [inline tags.html]: Would it be possible to fix the indentations on this page please?

File: _assets/stylesheets/_base.scss [hunk 2]
```diff
@@ -153,3 +153,11 @@ img {
   color: $white;
   background-color: $gray-darkest;
 }
+
+.text-center {
```
Comment (nielsenramon) [inline _assets/stylesheets/_base.scss]: I would prefer to not use this and add the properties to the right classes in modules/tags.

**Comment by nielsenramon** [comment]
@xRahul Nice! Once it's ready for merging I will merge it and style it a bit + add a link to the page.

File: tags.html (hunk 1 above)
Comment (xRahul) [reply tags.html]: done. There was a tabs/spaces mix up in the file. I converted them all to spaces.

File: _assets/stylesheets/_base.scss (hunk 2 above)
Comment (xRahul) [reply _assets/stylesheets/_base.scss]: In my experience, I have encountered the need to use text-center a lot, so I have this clreated for almost all of my projects. And sure, let me know what styling you end up with. I was never good at that part anyway.

Comment (nielsenramon) [reply _assets/stylesheets/_base.scss]: @xRahul I know but for now please keep it in one file. I will introduce some functionality classes for this later!

Comment (xRahul) [reply _assets/stylesheets/_base.scss]: Should I inline it till then? or another scss file that gets imported in light/dark?

Comment (nielsenramon) [reply _assets/stylesheets/_base.scss]: @xRahul I would just add the properties to the classes created in tags.scss to be honest!

Comment (xRahul) [reply _assets/stylesheets/_base.scss]: that's fine too. I'll update the pr

**Comment by xRahul** [comment]
As the commits increased a lot, I'll raise another one.

**Pull request closed without merging**